pyvista/plotting/prop3d.py

```diff
@@ -220,9 +254,24 @@ def orientation(self) -> Tuple[float, float, float]:  # numpydoc ignore=RT01
         return self.GetOrientation()
 
     @orientation.setter
-    def orientation(self, value: Tuple[float, float, float]):  # numpydoc ignore=GL08
+    def orientation(self, value: VectorLike[float]):  # numpydoc ignore=GL08
```

Comment:
The expression `VectorLike` would allow vectors of all dimensions. Do we need to confirm inside the method that it is 3-dimensional? Or we will not do it but do in `vtk`?

Reply:
I did a search and confirmed that there was only one other same one.
https://github.com/pyvista/pyvista/blob/ec1dea7d3057127343c198452ee18f8ba1b3bef9/pyvista/plotting/actor_properties.py#L53-L55

Reply:
Roger that. Let's leave it as is and discuss it in another PR.
```suggestion
    def orientation(self, value: Tuple[float, float, float]):  # numpydoc ignore=GL08
```